chain-alerter/src/main.rs

```diff
@@ -101,6 +101,11 @@ struct Args {
     /// Slack is enabled by default, and required a Slack OAuth secret file named `slack-secret`.
     #[arg(long, default_value = "true", action = ArgAction::Set)]
     slack: bool,
+
+    /// Enable or disable Subspace RPC connections.
+    /// Subspace RPCs are enabled by default.
+    #[arg(long, default_value = "true", action = ArgAction::Set)]
+    subspace: bool,
```

Comment:
Is the only utility of this flag for testing in CI? Having to maintain `RpcClientList` as an `Option` for CI seems a bit invasive.